src/utils/constants.js

```diff
@@ -1 +1 @@
-export const API_BASE_URL = 'https://wizeline-academy.cdn.prismic.io/api/v2';
\ No newline at end of file
+export const API_BASE_URL = "https://wizeline-academy.cdn.prismic.io/api/v2";
```

Comment:
nit: Consider storing your API url as an .env variable

https://create-react-app.dev/docs/adding-custom-environment-variables/

Reply:
I'm going to skip this for now.

Reply:
I'm skipping this for now